lib/compat/wordpress-6.7/block-templates.php

```diff
@@ -33,7 +32,7 @@ function wp_register_block_template( $template_name, $args = array() ) {
 	 * Unregister a template.
 	 *
 	 * @param string $template_name Template name in the form of `plugin_uri//template_name`.
-	 * @return true|WP_Error True on success, WP_Error on failure or if the template doesn't exist.
+	 * @return WP_Block_Template|WP_Error True on success, WP_Error on failure or if the template doesn't exist.
```

Comment:
I guess the `True on success` part need to change here and at the core PR.

Reply:
Good catch! Fixed in d5b7efe966f4b47223f67b493a47ea3a435e2b64.